plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -134,14 +134,18 @@ const CatalogPage: FC<{}> = () => {
           variant="info"
           message={
             <Typography>
-              <span role="img" aria-label="wave" style={{ fontSize: '125%' }}>
+              <span
+                role="img"
+                aria-label="wave"
+                style={{ fontSize: '125%', marginRight: 12 }}
```

Comment:
I'd move this into the `useStyles` block, and do `fontSize: '125%', marginRight: theme.spacing(2)`

Reply:
Yeah, good point. Fixed now.